server/routes/routerRequestHandlers/retrieveEducation.ts

```diff
@@ -16,31 +16,12 @@ const retrieveEducation = (educationAndWorkPlanService: EducationAndWorkPlanServ
         educationDto: await educationAndWorkPlanService.getEducation(prisonNumber, req.user.username),
       }
     } catch (error) {
-      res.locals.education = { ...gracefullyHandleException(error, prisonNumber), educationDto: undefined }
+      logger.debug('Error retrieving Education data', error)
```

Comment:
The code that calls the service method no longer needs to handle any exception (gracefully or otherwise 🤣 ) because it is now handled in the service method itself